internal/testutil/pkgbuilder/builder.go

```diff
@@ -539,6 +539,35 @@ func buildKptfile(pkg *pkg, pkgName string, reposInfo ReposInfo) string {
 	return result
 }
 
+func resolveRepoRef(repoRef string, reposInfo ReposInfo) string {
```

Comment:
It would be helpful to add godoc for all the 3 methods with example input and output.